src/doctypes.js

```diff
@@ -214,7 +284,7 @@ export const transactionsConn = {
   query: () =>
     Q(TRANSACTION_DOCTYPE)
       .UNSAFE_noLimit()
-      .include(['bills', 'account', 'reimbursements', 'recurrence']),
+      .include(['bills', 'account', 'reimbursements', 'recurrence', 'tags']),
```

Comment:
What are the impacts on the performance? Does the queries are slower?

Reply:
C'est dur à dire, la vitesse fluctue beaucoup selon les tests... mais non j'ai pas l'impression.